### ISSUE TITLE
Effect of normalisation?

### ISSUE THREAD

**Issue by kaikai23** [OutofAi/2D-Gaussian-Splatting#8]
Hi, really a great work for people to get a fist hand on gaussian splatting!

I have one question regarding the normalisation in function `generate_2D_gaussian_splatting`:
The kernel was first computed by `torch.exp(z)/ 2*pi*sqrt(det(covariance))`, and then normalised such that the max pixel value for it is 1. Since `torch.exp(z)` already satisfies that the max pixel value is 1, why do we bother to do this redundant computation?

As a simple test, I print the value for `kernel_normalized` and `torch.exp(z)`, and they look the same:
![image](https://github.com/OutofAi/2D-Gaussian-Splatting/assets/71677542/f5d62b0e-b565-4c7e-88c6-50f20a66615d)

**Comment by JiangBing123** [comment]
maybe the author write the full expression to make us understand the processing easier
